src/providers/staticfile.rs

```diff
@@ -0,0 +1,117 @@
+use super::Provider;
+use crate::nixpacks::{
+    app::{App, StaticAssets},
+    environment::Environment,
+    nix::pkg::Pkg,
+    phase::{BuildPhase, SetupPhase, StartPhase},
+};
+use anyhow::Result;
+use indoc::formatdoc;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Serialize, Deserialize, Default, Debug)]
+pub struct Staticfile {
```

Comment:
I'm not sure, should this be StaticFile?

Reply:
`Staticfile` is fine IMO.

Reply:
`Staticfile` is being treated as a single word [here](https://docs.cloudfoundry.org/buildpacks/staticfile/) I guess (like `Procfile`).